recipes/halotools/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set name = "halotools" %}
+{% set version = "0.5" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "7f5e8c3afcee35baa053a2c084d1b1301de70a285fce2391b7e46fedb53e041d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.9.*  # [not (win and py36)]
```

Comment:
>  You could end up in a situation where on non-windows and py36 systems, you build with 1.9 but then run with 1.12.

@synapticarbors that is fine and it is the goal here. `numpy` is backwards compatible and, as long as we build with an old enough `numpy`, we can run with any older version that does not break the ABI.

This is the same approach used to build wheels BTW.